tests/test_app/App/Controller/Admin/Management/PostsController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link      http://cakephp.org CakePHP(tm) Project
+ * @since     1.1.2
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Bake\Test\App\Controller\Admin\Management;
+
+use Cake\Controller\Controller;
+
+/**
+ * Posts Controller class.
+ *
+ * For testing prefix routing.
+ */
+class PostsController extends Controller
+{
+    /**
+     * index action
+     *
+     * @return void
```

Comment:
We should probably correct those to allow for response returns. void would not be quite correct.